js/components/sidebar-menu.js

```diff
@@ -166,38 +166,43 @@ export default class SidebarMenu extends PureComponent {
           {
             OPTION_ENABLED.latLongLines &&
             <ListCheckbox caption='Latitude and longitude lines' legend='Geographic coordinate system'
-              checked={options.renderLatLongLines} onChange={this.toggleLatLongLines} />
+              checked={options.renderLatLongLines} onChange={this.toggleLatLongLines} className='list-item'/>
```

Comment:
I tried moving this file to `css-modules`, and using class names like `{css.listItem}` but I couldn't get them picked up for some reason. I also tried applying themes instead of classes here, but I couldn't get them picked up either.

Reply:
I guess you can find working examples both of CSS modules and React-toolbox themes, so they have to work. Did you put the file in `css-modules` dir? That's the webpack rule that enables modules. But this file doesn't use modules yet, so it's fine to leave it like that.